### PR TITLE
Fix broken link in resources documentation

### PR DESCRIPTION
**Pull request by gemmahou**
### Change description

Fixes b/320298417, b/320297276

### Tests you have done

- [x] Run `make ready-pr` to ensure this PR is ready for review.
- [N/A] Perform necessary E2E testing for changed resources.

### CODE REVIEW

**Comment by gemmahou** [comment]
cc: @tarynlucas

File: scripts/generate-google3-docs/resource-reference/templates/sql_sqlsslcert.tmpl [hunk 1]
```diff
@@ -27,7 +27,7 @@
 </tr>
 <tr>
 <td>{{"{{gcp_name_short}}"}} REST Resource Documentation</td>
-<td><a href="/sql/docs/reference/admin-api/rest/v1beta4/sslCerts">/sql/docs/reference/admin-api/rest/v1beta4/sslCerts</a></td>
+<td><a href="/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts">/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts</a></td>
```
Comment (maqiuyujoyce) [inline scripts/generate-google3-docs/resource-reference/templates/sql_sqlsslcert.tmpl:30]: Looks like other SQL resource doc all used the postgres links:

https://cloud.google.com/config-connector/docs/reference/resource-docs/sql/sqluser
https://cloud.google.com/config-connector/docs/reference/resource-docs/sql/sqldatabase
https://cloud.google.com/config-connector/docs/reference/resource-docs/sql/sqlinstance

So we can probably do either of the following:
1. Also use the postgres link here for consistency, or
2. Add the links for all the SQL versions in all the SQL resource docs similar to what we did in the Logging resources (https://cloud.google.com/config-connector/docs/reference/resource-docs/logging/logginglogbucket), i.e. 
```
<pre><a href="/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts">/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts</a></pre>
<pre><a href="/sql/docs/postgres/admin-api/rest/v1beta4/sslCerts">/sql/docs/postgres/admin-api/rest/v1beta4/sslCerts</a></pre>
<pre><a href="/sql/docs/sqlserver/admin-api/rest/v1beta4/sslCerts">/sql/docs/sqlserver/admin-api/rest/v1beta4/sslCerts</a></pre>
```

Comment (gemmahou) [reply scripts/generate-google3-docs/resource-reference/templates/sql_sqlsslcert.tmpl:30]: It won't affect anything, as the doc for these three databases are the same. So I'll just use postgres for consistency and make sure the link works.

**Pull request merged**